pkg/cmd/repo/license/license.go

```diff
@@ -0,0 +1,20 @@
+package license
+
+import (
+	cmdList "github.com/cli/cli/v2/pkg/cmd/repo/license/list"
+	cmdView "github.com/cli/cli/v2/pkg/cmd/repo/license/view"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdLicense(f *cmdutil.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "license <command>",
+		Short: "View available repository license templates",
```

Comment:
Figuring out how to explain a commandset is a bit rough and none of the other examples really line up.  Since we're only making license information available, I thought maybe the following would be short and sweet:

```suggestion
		Short: "Explore repository licenses",
```